### PR TITLE
User can create a trail

### PR DESCRIPTION
**Pull request by lucamarial**
[PT-link](https://www.pivotaltracker.com/story/show/169699081)

## This PR contains
```
As a user
In order to share my favourite trail
I need a website where I can create a trail and add information about it
```

## PR content
- CreateTrail component
- CreateTrailForm components
  - Input fields
  - Button
  - Rating slider
  - TextArea
- Integration test for creating a Trail
- Fixture files for response Messages
- TrailsData module

### CODE REVIEW

File: src/Modules/TrailsData.js [hunk 1]
```diff
@@ -13,4 +13,21 @@ const getTrails = async () => {
   }
 }
```
Comment (faraznaeem) [inline src/Modules/TrailsData.js]: Please rename this file to `trailsData.js`

File: cypress/fixtures/cannot_create_trail.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "error_message": "Description is too short (minimum is 15 characters)"
```
Comment (piavW) [inline cypress/fixtures/cannot_create_trail.json]: it is `Please add more content`

**Comment by faraznaeem** [comment]
Please add a screenshot

File: src/App.jsx [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
+import CreateTrail from './Components/CreateTrail'
 
 const App = () => {
   return (
     <>
       <Route exact path='/' component={LandingPage} />
+      <Route exact path='/create' component={CreateTrail} />
     </>
   )
 }
```
Comment (piavW) [inline src/App.jsx:13]: extra space?

**Review by piavW — CHANGES_REQUESTED** [review]
Add screenshot.

**Review by piavW — APPROVED** [review]

File: src/Modules/trailsData.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+const apiUrl = 'http://localhost:3000/v1/'
+
+const getTrails = async () => {
+  try {
+    let response = await axios.get(apiUrl + 'trails')
+    return response.data.trails
+  } catch(error) {
+    return {
+      error: error.response.data.error
```
Comment (DIzosimov) [inline src/Modules/trailsData.js:11]: are you sure its .error if you have .error_message in the actual function in component, or does the response look `error.response.data.error.error_message` or `error.response.data.error_message` if second one then remove the last .error

**Review by jonas-masharqa — APPROVED** [review]

**Review by DIzosimov — APPROVED** [review]

**Review by AlexSaxena — APPROVED** [review]

**Review by DIzosimov — COMMENTED** [review]
Good

**Pull request merged**